float32 duration_sec
---
bool success
string message